api/FileSystemHandle.json

```diff
@@ -113,6 +113,39 @@
           }
         }
       },
+      "move": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/FileSystemHandle/move",
+          "support": {
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": "mirror",
+            "edge": "mirror",
+            "firefox": {
+              "version_added": "111"
+            },
+            "firefox_android": "mirror",
+            "ie": {
+              "version_added": false
+            },
+            "oculus": "mirror",
+            "opera": "mirror",
+            "opera_android": "mirror",
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": "mirror",
+            "samsunginternet_android": "mirror",
+            "webview_android": "mirror"
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": false,
+            "deprecated": false
+          }
+        }
+      },
```

Comment:
@queengooborg FYI support for this was added in FF111 in https://bugzilla.mozilla.org/show_bug.cgi?id=1811001. However appears to be explicitly non-standard at this point as it is in a "PR" state - see the big banner when you access the URL https://whatpr.org/fs/10.html#api-filesystemhandle

Upshot, I've added it because that makes the compatibility story easier to track later, but there is no spec URL and it is marked as experimental and non-standard.

@saschanaz Added with above caveats ^^^. I don't plan to add documentation for this right now; any idea on when this PR is likely to go in?

Reply:
I have no idea, maybe @janvarga knows?

Reply:
@saschanaz - cool - not urgent - just useful so we get a ping when it becomes standard.

BTW, thanks for all your help and advice on this. I will do better next time.

Reply:
No problem, you're welcome 👍